modules/pushapk_scriptworker/manifests/settings.pp

```diff
@@ -259,6 +272,29 @@
                     'skip_check_same_locales' => true,
                     'skip_checks_fennec' => true,
                 },
+
+                'fenix-nightly' => {
+                    'has_nightly_track' => true,
+                    'service_account' => $google_play_config['fenix-nightly']['service_account'],
+                    'certificate' => $google_play_config['fenix-nightly']['certificate_target_location'],
+                    'update_google_play_strings' => false,
+                    'digest_algorithm' => 'SHA-256',
+                    'expected_package_names' => ['org.mozilla.fenix'],
+                    'skip_check_multiple_locales' => true,
+                    'skip_check_same_locales' => true,
+                    'skip_checks_fennec' => true,
+                },
+                'fenix-beta' => {
+                    'has_nightly_track' => true,
```

Comment:
Do we want to use a different name for the testing branch?

Reply:
That's a really good question, I'll confirm with Sylvestre :)

Reply:
Sylvestre recommended the term `internal`, so I'm going to need to provide a PR to `pushapkscript`